addons/controllers/antrea/antreaconfig_controller.go

```diff
@@ -26,16 +29,35 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	nsxoperatorapi "github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
 	addontypes "github.com/vmware-tanzu/tanzu-framework/addons/pkg/types"
 
 	cutil "github.com/vmware-tanzu/tanzu-framework/addons/controllers/utils"
 	addonconfig "github.com/vmware-tanzu/tanzu-framework/addons/pkg/config"
 	"github.com/vmware-tanzu/tanzu-framework/addons/pkg/constants"
 	"github.com/vmware-tanzu/tanzu-framework/addons/pkg/util"
 	"github.com/vmware-tanzu/tanzu-framework/addons/predicates"
-	cniv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/addonconfigs/cni/v1alpha1"
+	cniv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/addonconfigs/cni/v1alpha2"
```

Comment:
Should we change this `cnialpha1` to `cniv1alpha2`? Same comment for `cni/v1alpha2` import in file addons/controllers/antrea/antreaconfig_util.go.

Reply:
thanks, done